common/test/acceptance/tests/studio/test_studio_home.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
         self.auth_page = AutoAuthPage(self.browser, staff=True)
         self.dashboard_page = DashboardPage(self.browser)
 
-    @flaky  # TODO: SOL-430
+    @flaky(max_runs=30, min_passes=30)
```

Comment:
@bdero I know - the goal here was to force jenkins to run it multiple times and fail the run if the tests fails once (see comments on SOL ticket) - so I've added this instead of removing `flaky` decorator to make sure it is actually not flaky anymore and I'll remove it now.